src/field_theory/intermediate_field.lean

```diff
@@ -286,4 +286,29 @@ instance has_lift2 {F : intermediate_field K L} :
 
 end tower
 
+section finite_dimensional
```

Comment:
This looks like it can be an easy PR on its own.

Reply:
#4828